testproject/Assets/Scripts/ConnectionModeScript.cs

```diff
@@ -28,15 +28,12 @@ public class ConnectionModeScript : MonoBehaviour
 
     private CommandLineProcessor m_CommandLineProcessor;
 
-    private bool m_UsingRelay = false;
-
-#if ENABLE_RELAY_SERVICE
+    //[NSS] This is never used?  Do we need it still?
     [SerializeField]
```

Comment:
It will be used when relay fixing an issue in their SDK

Reply:
Well, we can just leave that as it is since there doesn't appear to be any relay specific properties being used.
I tried to keep the defines limited to only the areas of the code that had dependencies to the relay portion of the UnityTransport.